src/Microsoft.DotNet.Build.Tasks/PackageFiles/tool-runtime/project.json

```diff
@@ -3,38 +3,27 @@
     "dnxcore50": {
```

Comment:
That would be a breaking change because the publish we are doing is for dnxcore50 which is why I added [this here](https://github.com/dotnet/buildtools/pull/706/files#diff-05eb29e66e87abd2de147e792d51d6b3R26). Ultimately I was going to have  it as you suggest, just thought of staging the changes of first changing project.jsons and then changing init-tools scripts, but I can change it to that if you want